### ISSUE TITLE
Multi-community support ("multireddit")

### ISSUE THREAD

**Issue by ShadowJonathan** [mlemgroup/mlem#122]
Implementing the "multireddit" functionality in the app, for probably a few reasons;

One is to group together multiple communities into one pagination, a-la multireddit.

Another is due to the nature of Lemmy's federation, where multiple communities under the same name could be grouped together quickly to "catch" every single post from every single instance.

However, this would have to rely on the Lemmy API to get it done, and so unless we want to implement everything client-side, this issue is blocked on https://github.com/LemmyNet/lemmy/issues/818

**Comment by Sjmarf** [comment]
Closing in favor of using the Lemmy/PieFed implementation #2099
